dynamic-config/model/src/main/java/org/terracotta/dynamic_config/api/service/ConfigPropertiesTranslator.java

```diff
@@ -111,19 +112,32 @@ public Properties load(Path configFile) {
    * Warning: caller who created the stream is also responsible for closing it
    */
   public Properties convert(Reader stream) throws IOException {
-
     // Read each line in the file stream, ignoring all line comments
     // Line format is: [[stripe|node :]<stripe_or_node_name>:]<setting>=<value>
-    // splits happen at '='
+    // splits only happen at '='
+    // line continuations are supported
 
     errors.clear();
     inputConfigSettings.clear();
     try (BufferedReader reader = new BufferedReader(stream)) {
-      reader.lines()
-          .map(String::trim)
-          .filter(line -> !line.isEmpty() && !line.startsWith("#") && !line.startsWith("!"))
-          .map(line -> line.split(EQUALS))
-          .forEach(splits -> inputConfigSettings.put(splits[0].trim(), splits.length > 1 ? splits[1].trim() : ""));
+      List<String> lines = reader.lines()
+        .map(String::trim)
+        .filter(line -> !line.isEmpty() && !line.startsWith("#") && !line.startsWith("!"))
+        .collect(Collectors.toList());
+      Iterator<String> itr = lines.iterator();
+      String line = "";
+      while (itr.hasNext()) {
+        String thisLine = itr.next();
+        char c = thisLine.charAt(thisLine.length() - 1);
+        if (c == '\\') {
+          line += thisLine.substring(0, thisLine.length()-1).trim();
+        } else {
+          line += thisLine;
+          String[] splits = line.split("=");
+          inputConfigSettings.put(splits[0], splits.length > 1 ? splits[1].trim() : "");
```

Comment:
this is where I would trim: at the very end, when creating the properties
```suggestion
          inputConfigSettings.put(splits[0].trim(), splits.length > 1 ? splits[1].trim() : "");
```

Reply:
updated as suggested.